configs/default-mappings.json

```diff
@@ -1,13 +1,43 @@
 {
     "mappings": {
-        "dynamic": false,
+        "dynamic": "true",
```

Comment:
Can the dynamic be strict instead of true, meaning new fields will be created when indexing and will reject any cat item not adhering to the mapping